### ISSUE TITLE
Can't delete OrderNote

### ISSUE THREAD

**Issue by dkainth** [XiaoFaye/WooCommerce.NET#404]
Wordpress version: 5.2.3
WooCommerce version: 3.70
WooCommerce.NET version: 0.78

Unable to delete a order note:
`var note = _wcObject.Order.Notes.Delete(noteId, orderId, true);`
(setting force to true as that is indicated in the [API documents](https://woocommerce.github.io/woocommerce-rest-api-docs/#delete-an-order-note))

Error message:
`'{"code":"woocommerce_rest_trash_not_supported","message":"Webhooks do not support binning.","data":{"status":501}}'`

**Comment by XiaoFaye** [comment]
It seems WooCommerce doesn't support force delete an order note anymore, but the API document is not updated. Can you confirm this with WooCommerce?

**Comment by dkainth** [comment]
Sorry I should have tested that and included that info in my original post.

Calling the Woocommerce API directly using Postman this works:

`https://mydomain.co.uk/wp-json/wc/v2/orders/46445/notes/12385?force=true`

but this fails:
`https://mydomain.co.uk/wp-json/wc/v2/orders/46445/notes/12386`

and the error message is :
```
{
    "code": "woocommerce_rest_trash_not_supported",
    "message": "Webhooks do not support binning.",
    "data": {
        "status": 501
    }
}
```

so I think the call that the NET library is making, is omitting the param ?force=true. Can check tomorrow to confirm that is the case, but that is what it currently looks like.

Thanks

**Comment by XiaoFaye** [comment]
You are right, this has been omitted, will fix this in the next version. For now, you can use RestAPI.DeleteRestful to manually send a request.

**Comment by dkainth** [comment]
Thanks, will do

**Comment by XiaoFaye** [comment]
@dkainth Hi, this has been fixed in version 0.7.9.
